Example/EmissionTests/TestApps.js

```diff
@@ -1,24 +1,21 @@
-/* @flow */
-'use strict';
-
-import Emission from 'emission';
-import React from 'react';
-import ReactNative from 'react-native';
-const { View } = ReactNative;
-const { TestModule } = ReactNative.NativeModules;
+import Emission from 'emission'
+import React from 'react'
```

Comment:
nah, just checked, this doesn't need to change - it's still a JS file, and should stay one probably.  It's only referenced by the iOS Sim Test Runner.